packages/testcontainers/src/generic-container/generic-container.test.ts

```diff
@@ -123,6 +123,19 @@ describe("GenericContainer", () => {
     expect(output).toEqual(expect.stringContaining("/tmp"));
   });
 
+  it("should set platform", async () => {
+    // This test uses an older version of cristianrgreco/testcontainer as all the other tests are using the latest version and the code doesn't pull the image again when the platform differs at this time
```

Comment:
https://node.testcontainers.org/features/containers/#with-a-pull-policy

Reply:
Thanks. I have updated it.